lib/stdlib/fields.juttle

```diff
@@ -0,0 +1,74 @@
+// Manipulate fields within a stream with these field reducers
+
+// Return an array of last N values of a field
+//
+// Parameters:
+//
+//   * field: ("value") field to tail
+//   * length: (1) number of recent values to retain
+export reducer tail(field, length=1) {
+    var data = [];
+
+    function update() {
+        if (*field != null) { 
```

Comment:
Do we have a different way of detecting the field isn't in the data vs the field is but set to null ? Because the problem here is points that don't have the field desired shouldn't count in the reduction right ?